README.md

```diff
@@ -2,9 +2,9 @@
 
 > Enables static websites deployment to Amazon S3 website buckets using Capistrano.
 
-[![Build Status](https://img.shields.io/travis/hooktstudios/capistrano-s3/master.svg)](https://travis-ci.org/hooktstudios/capistrano-s3)
-[![Dependency Status](https://img.shields.io/gemnasium/hooktstudios/capistrano-s3.svg)](https://gemnasium.com/hooktstudios/capistrano-s3)
-[![Code Climate](https://img.shields.io/codeclimate/github/hooktstudios/capistrano-s3.svg)](https://codeclimate.com/github/hooktstudios/capistrano-s3)
+[![Build Status](https://img.shields.io/travis/capistrano-s3/capistrano-s3/master.svg)](https://travis-ci.org/capistrano-s3/capistrano-s3)
```

Comment:
I think we could now directly link travis they provide an SVG https://travis-ci.org/capistrano-s3/capistrano-s3.svg?branch=master